### ISSUE TITLE
Credentials are only filled when leaving and reentering browser tab

### ISSUE THREAD

**Issue by synthead** [marius-wieschollek/passwords-webextension#157]
### System Information
- Extension Version: 2.1.0
- App Version: 2021.2.0
- Browser and Version: Firefox 86.0
- Client OS and Version: Arch Linux (rolling release, no version)

### Steps to reproduce
1.  Use these settings: 
![image](https://user-images.githubusercontent.com/820984/109371040-d969a780-7857-11eb-943a-fe401d50f0bf.png)

2.  Navigate to a site with a login form
3.  Observe that the form is not filled with credentials from the extension

### Workaround
1. Switch browser tabs to anything else
1. Return to the tab with the login form
1. Observe that the credentials are now filled

### Expected result
Credentials are filled after the page loads

### Actual result
Credentials are not filled after the page loads

### Browser log
<details>

```
Use of nsIFile in content process is deprecated. 15 NetUtil.jsm:253:8
Could not establish connection. Receiving end does not exist. Error: Could not establish connection. Receiving end does not exist.
    _sendMessage moz-extension://04812848-3e5a-4e14-87c5-2525d1f59fdd/js/background.js:1
 
Object { details: {…}, error: {…} }
 _sendMessage@moz-extension://04812848-3e5a-4e14-87c5-2525d1f59fdd/js/background.js:1:11110
background.js:1:5856
Could not establish connection. Receiving end does not exist. Error: Could not establish connection. Receiving end does not exist.
    _sendMessage moz-extension://04812848-3e5a-4e14-87c5-2525d1f59fdd/js/background.js:1

```

</details>

**Comment by flo-mic** [comment]
This issue shoud be solved. The related pull request was merged this morning, see #152

**Comment by marius-wieschollek** [comment]
This should now be solved in the nightly release.

**Comment by synthead** [comment]
Looks like it's solved in 2.1.1!  Thanks, y'all!

**Comment by marius-wieschollek** [comment]
Thanks to @flo-mic for the fix
